examples/web/README.md

```diff
@@ -0,0 +1,165 @@
+# Node.js Client SDK
+
+_Read this in other languages_: [日本語](README.ja.md)
+
+<br>
+
+## Example Requirements
+
+- Node version 14 or higher is required
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+
+To run any of the examples you will need to install the dependencies once first:
+
+```bash
+npm install
+```
+
+## Running the Basic Example
+
+```bash
+# Run example code
+MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run example
+```
+
+Example Code: [basic.ts](basic.ts)
+
+## Running the Advanced Example
+
+```bash
+# Run example code
+MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run advanced
+```
+
+Example Code: [advanced.ts](advanced.ts)
+
+## Running the Dictionary Example
+
+This example demonstrates how to use the dictionary data type.
+
+```bash
+# Run example code
+MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run dictionary
+```
+
+Example Code: [dictionary.ts](dictionary.ts)
+
+## Running the Pubsub Example
```

Comment:
i think we can remove the pubsub, loadgen, and coalescing info from this readme?